migrations/sql/V2021.09.02.03.00__add_phone_and_email_columns_to_party.sql

```diff
@@ -0,0 +1,10 @@
+
+ALTER TABLE party ADD COLUMN phone_no_sec character varying(12) NULL;
```

Comment:
Can you add "IF NOT EXISTS" to each of these? Migrations are generally not re-runnable but sometimes this causes issues locally. We also don't want to enforce character lengths, or any other types of similar constraints, at the DB level; something that was decided upon very early in the project. We only do this for some things like code tables now.

Reply:
You might want to take a look at \mds\migrations\sql\V2021.02.24.12.59__update_party_and_address_tables_remove_length_limitations.sql
for avoiding character lengths in the phone numbers.